runtime/src/server.ts

```diff
@@ -21,6 +21,8 @@ export type Headers = object;
 
 export type Query = object;
 
+export type RequestContext = any;
```

Comment:
Don't you want to make it more restricted?

Reply:
This is the widest possible type for the request context, from which the real request context needs to be extended. Due to maintaining backwards compatibility it needs to be pretty wide. The real request context type is defined in the project using the oats-koa-adapter.